sci-electronics/yosys/yosys-0.40-r1.ebuild

```
@@ -25,16 +25,15 @@ RDEPEND="
 DEPEND="${RDEPEND}"
 BDEPEND="dev-vcs/git"
 
-QA_PRESTRIPPED="
-	/usr/bin/yosys-filterlib
-	/usr/bin/yosys-abc
-"
-
 src_prepare() {
 	mv "${WORKDIR}/abc-${ABC_GIT_COMMIT}" "${S}"/abc || die
 	default
 }
 
-src_install() {
-	emake DESTDIR="${D}" PREFIX='/usr' install
+src_configure() {
+	cat <<-__EOF__ >> Makefile.conf
```

```suggestion
	cat <<-__EOF__ >> Makefile.conf || die
```

LGTM, 
Thanks for your improvements to the ebuild, it makes the yosys package look better, I'll commit to the dev branch.

The GURU project cannot be merged directly from github because it is just a mirror. Next time you can try to follow the instructions at https://wiki.gentoo.org/wiki/Project:GURU/Information_for_Contributors.